### PR TITLE
Styling development

### PR DESCRIPTION
**Pull request by jnnybk**
1) Remove Toggle function and change to display all information upon load. 
2) Add light / dark mode. 
3) Add fade in upon load. 
4) Add border around comment box.

### CODE REVIEW

**Comment by jnnybk** [comment]
I deployed on my StylingDevelopment branch to show my progress so far: http://baikj-step-2020.appspot.com/

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,15 +7,51 @@
...
+            </ul>
+          <h3>Songs that she's currently listening to!</h3>
+            <div> 
+              <button class="song-buttons" id="prev"><</button>
+              <a href= "" id="wrapper">
+                <img name="albumSlide" width="200" height="200" />
+              </a>
+              <button class="song-buttons" id="next">></button>
+            </div>
+          <h3><a style="color: #4a4e69;text-decoration:none;" href="/images/Jenny_Baik_Resume.pdf">Click for Resume!</a></h3>
```
Comment (belraquib) [inline portfolio/src/main/webapp/index.html:50]: here and elsewhere; can you set the style in css?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -138,3 +122,21 @@ function deleteComment(comment) {
...
   document.getElementById('maxNum').addEventListener('click', displayComments);
 });
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('mode').addEventListener('click', toggleMode);
+})
+
+function toggleMode() {
+  let element = document.body;
+  element.classList.toggle("dark-mode");
```
Comment (belraquib) [inline portfolio/src/main/webapp/script.js:132]: you can instead write


document.boddy.classList.toggle("dark-mode")

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -138,3 +122,21 @@ function deleteComment(comment) {
...
+
+  let anchors = document.getElementsByTagName("a");
```
Comment (belraquib) [inline portfolio/src/main/webapp/script.js:134]: you can do this in css;


body.dark-mode a {
   color: rgb(..)
}
body a {
    color: white
}

**Pull request merged**